Backend/lib/scheduler/src/Lib/Scheduler/App.hs

```diff
@@ -46,26 +49,30 @@ runSchedulerService SchedulerConfig {..} handle_ = do
   loggerEnv <- prepareLoggerEnv loggerConfig hostname
   esqDBEnv <- prepareEsqDBEnv esqDBCfg loggerEnv
   coreMetrics <- Metrics.registerCoreMetricsContainer
-  hedisEnv <- connectHedis hedisCfg (\k -> hedisPrefix <> ":" <> k)
-  hedisNonCriticalEnv <- connectHedis hedisNonCriticalCfg (\k -> hedisPrefix <> ":" <> k)
+  hedisEnv <- connectHedis hedisCfg id
+  hedisNonCriticalEnv <- connectHedis hedisNonCriticalCfg id
   hedisNonCriticalClusterEnv <-
     if cutOffHedisCluster
       then pure hedisNonCriticalEnv
-      else connectHedisCluster hedisNonCriticalClusterCfg (\k -> hedisPrefix <> ":" <> k)
+      else connectHedisCluster hedisNonCriticalClusterCfg id
   hedisClusterEnv <-
     if cutOffHedisCluster
       then pure hedisEnv
-      else connectHedisCluster hedisClusterCfg (\k -> hedisPrefix <> ":" <> k)
+      else connectHedisCluster hedisClusterCfg id
   metrics <- setupSchedulerMetrics
   isShuttingDown <- mkShutdown
-
+  let schedulerType = RedisBased
+  let groupName = "myGroup"
+  let setName = "Scheduled_Jobs"
+  let streamName = "Available_Jobs"
```

Comment:
shouldn't these values be configurable in dhall?

Reply:
`runSchedulerService` doesn't have dhall config